public/data/atharvamogade.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Atharva Mogade",
+    "bio": "An Aspiring Web Developer, Open Sourse and UI UX Enthusiast.",
+    "avatar": "https://github.com/atharvamogade.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/atharvamogade",
+        "icon": "github"
+      },
+      {
+        "name": "Conenct with me on Twitter",
+        "url": "https://twitter.com/atharvamogade",
+        "icon": "twitter"
+      },
+      {
+        "name": "Conenct with me on LinkedIn",
+        "url": "https://linkedin.com/in/atharvamogade",
+        "icon": "send"
+      }
+    ]
+  }
```

Comment:
GitHub is highlighting that this needs an extra blank line - we can merge and this can be fixed later